src/sources/Blheli/escsAtmel.json

```diff
@@ -0,0 +1,51 @@
+{

```

Comment:
Maybe it would be nicer to have a sperate sub dir for the escs?

* escs/Atmel.json
* escs/Silabs.json

Reply:
They are also different classes, so could have their own dir entirely, but wasn't sure since they are some what related but also not really used atm.